Source/WebCore/Modules/applicationmanifest/ApplicationManifestParser.cpp

```diff
@@ -173,6 +174,32 @@ URL ApplicationManifestParser::parseStartURL(const JSON::Object& manifest, const
     return startURL;
 }
 
+ApplicationManifest::Direction ApplicationManifestParser::parseDir(const JSON::Object& manifest)
+{
+    auto value = manifest.getValue("dir"_s);
+    if (!value)
+        return ApplicationManifest::Direction::Auto;
```

Comment:
Can put `using enum ApplicationManifest::Direction` at the top of the function and make the entire thing more terse without losing clarity.